controllers/reviews.controller.js

```diff
@@ -40,3 +42,15 @@ exports.getCommentsByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = async (req, res, next) => {
+  try {
+    const commentObj = req.body;
+    const review_id = req.params;
+
+    const comment = await insertComment(review_id, commentObj);
+    res.status(201).send({ comment });
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
If you're going to go with async/await, make sure you refactor all your functions at some point but ill let it slide for now